src/tls.c

```diff
@@ -4866,21 +4864,21 @@ int TLSX_ValidateSupportedCurves(WOLFSSL* ssl, byte first, byte second) {
 #endif
     {
         key = 1;
-        ssl->ecdhCurveOID = defOid;
+        cs->ecdhCurveOID = defOid;
```

Comment:
ssl->ecdhCurveOID used to get updated.
I can't see that happening in VerifyServerSuite anymore.
Is this intentional?